src/components/Results.tsx

```diff
@@ -1,31 +1,58 @@
-import { Grid } from '@mui/material';
+import { Box, Container, Grid, Pagination } from '@mui/material';
 import ResultCard from './ResultCard';
 import { Result, ResultList } from '../types/result';
+import { useState } from 'react';
 
-type ResultsProps = {
+interface ResultsProps {
```

Comment:
How come this was changed from `type` to `interface`?

Reply:
no reason, really;)